From: Branen Salmon <bugs@nenarb.net>
Date: Thu, 3 Oct 2019 16:58:16 -0400
Subject: Don't load external assets in HTML docs

For user privacy, we don't want to automatically load images or
iframes hosted on external servers when the user is browsing local
documentation.
---
 docs/source/_templates/sidebar.html | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/docs/source/_templates/sidebar.html b/docs/source/_templates/sidebar.html
index ec119ee..e0da91f 100644
--- a/docs/source/_templates/sidebar.html
+++ b/docs/source/_templates/sidebar.html
@@ -1,14 +1,3 @@
-<p class="logo">
-  <a href="{{ pathto(master_doc) }}">
-    <img class="logo" src="http://scorreia.com/images/software/2.jpg" title="written with a few of these"/>
-  </a>
-</p>
-
-<p>
-  <iframe src="http://ghbtns.com/github-btn.html?user=sergiocorreia&amp;repo=panflute&amp;type=watch&amp;count=true&amp;size=large"
-    allowtransparency="true" frameborder="0" scrolling="0" width="200px" height="35px"></iframe>
-</p>
-
 <p>Panflute is a Python package to easily write Pandoc filters.</p>
 <p>It is pythonic and comes with batteries included.</p>
 
@@ -16,8 +5,5 @@
 
 <p><a href="https://github.com/sergiocorreia/panflute/issues">Report issues or send suggestions</a>.</p>
 
-<p><iframe src="http://ghbtns.com/github-btn.html?user=sergiocorreia&amp;type=follow&amp;count=false"
-  allowtransparency="true" frameborder="0" scrolling="0" width="200" height="20"></iframe></p>
-
 <p>More projects by <a href="http://scorreia.com/">Sergio Correia</a> <a href="http://scorreia.com/software/">here</a>.</p>
 
